tests/test_importers_multi.py

```
@@ -209,6 +385,116 @@ def test_import_chatgpt_tool_call_pairing(tmp_path):
+        }
+    ]
+    (export_dir / "conversations.json").write_text(json.dumps(conversations), encoding="utf-8")
+    out_dir = tmp_path / "force_out"
+    first = import_chatgpt_export(
+        export_dir,
+        output_dir=out_dir,
+        collapse_threshold=10,
+        html=False,
+        html_theme="light",
+    )
+    assert first and not first[0].skipped
```

Ruff (0.15.9)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)